### PR TITLE
push: Fix accidental rename of `-e` (emulated) option

### PR DESCRIPTION
**Pull request by pdcastro**
`-e` option was accidentally renamed to `-f` during oclif conversion in #1960 / CLI v12.9.7.

Change-type: patch

### CODE REVIEW

File: lib/actions-oclif/push.ts [hunk 1]
```diff
@@ -121,8 +121,10 @@ export default class PushCmd extends Command {
 			char: 's',
 		}),
 		emulated: flags.boolean({
-			description: 'Force an emulated build to occur on the remote builder',
-			char: 'f',
+			description: stripIndent`
+				Don't use native ARM servers; force QEMU ARM emulation on Intel x86-64
+				servers during the image build (balenaCloud).`,
+			char: 'e',
```
Comment (pdcastro) [inline lib/actions-oclif/push.ts:127]: Reverting `'e'` as it was until #1960 and as documented online, e.g.: https://www.balena.io/docs/learn/deploy/deployment/#--emulated--e

While at it, I thought of rewording the option description for additional clarity and information.

File: lib/utils/compose.js [hunk 2]
```diff
@@ -17,90 +17,7 @@
 
 import * as path from 'path';
 import { ExpectedError } from '../errors';
-import { getChalk, stripIndent } from './lazy';
-
-export function appendOptions(opts) {
-	return opts.concat([
```
Comment (pdcastro) [inline lib/utils/compose.js:23]: I gather this code is no longer used after the conversion of the `build` and `deploy` commands to oclif.

File: lib/utils/docker-js.js [hunk 3]
```diff
@@ -20,92 +20,6 @@
...
 import { ExpectedError } from '../errors';
 
-// Use this function to seed an action's list of capitano options
-// with the docker options. Using this interface means that
-// all functions using docker will expose the same interface
-//
-// NOTE: Care MUST be taken when using the function, so as to
-// not redefine/override options already provided.
-export const appendConnectionOptions = (opts) =>
-	opts.concat([
```
Comment (pdcastro) [inline lib/utils/docker-js.js:30]: I gather this code is no longer used after the conversion of the `build` and `deploy` commands to oclif.

**Review by srlowe — APPROVED** [review]

**Comment by pdcastro** [comment]
@balena-ci rebase

**Pull request merged**